libs/expo/betterangels/src/lib/providers/featureFlag/featureFlagProvider.tsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useMemo, useState } from "react";
+import { FeatureFlagContext, FeatureFlags } from "./featureFlagContext";
+import { useGetFeatureFlagsQuery } from "./__generated__/featureFlagProvider.generated"
+
+
+interface FeatureFlagProviderProps {
+    children: React.ReactNode;
+}
+
+export const FeatureFlagProvider: React.FC<FeatureFlagProviderProps> = ({ children }) => {
+    const [featureFlags, setFeatureFlags] = useState<FeatureFlags>({});
+    const { data, error } = useGetFeatureFlagsQuery();
+
+    useEffect(() => {
+        if (data?.featureControls?.flags) {
```

Comment:
perhaps we should reset featureFlags if there's no data to be safe.
so something like this:
```javascript
if (!data?.featureControls?.flags) {
   setFeatureFlags({});

   return 
}

const flags = ....

```